cmd/cdi/main_test.go

```diff
@@ -3,15 +3,22 @@ package main_test
 import (
 	"testing"
 
+	"github.com/rendon/testcli"
 	"github.com/stretchr/testify/assert"
 )
 
-// TODO: Figure out how to test main finction
-// func TestMain(m *testing.M) {
-// 	exitCode := m.Run()
+func TestCommand(t *testing.T) {
+	c := testcli.Command("cdi")
```

Comment:
This just runs the binary - must be in the $PATH - So you might need to `export PATH=$PATH:$GOPATH/bin`